CHANGES

```diff
@@ -5,6 +5,7 @@
 - API Change: Net#openURI now returns a boolean to indicate whether the uri was actually opened.
 - API Change: DefaultShader now always combines material and environment attributes
 - API Change: Added ShapeRenderer constructor to pass a custom shader program to ImmediateModeRenderer20.
+- API Change: Added ClasspathFileHandleResolver for resolving assets via Gdx.files.classpath(string).
```

Comment:
Yes, @NathanSweet also suggested that recently. Seems like a reasonable approach that will make it easier to grep those particular changes at a glance.